draftjs_exporter/entities.py

```diff
@@ -27,23 +27,12 @@ def render(self, props):
 
 
 class Link:
-    attributes = ['url', 'rel', 'target', 'title']
-
-    @staticmethod
-    def is_valid_attribute(key):
-        # TODO How much do we need to whitelist / blacklist attributes?
-        valid_data_attr = key.startswith('data-') and len(key) > 5 and key.islower()
-        return key in Link.attributes or valid_data_attr
-
     def render(self, props):
```

Comment:
@loicteixeira I have removed this validation completely, as I don't think it should be handled for a single entity anyway. It should either be the whole exporter or nothing.

I'll likely also remove this whole `entity` file (at least from the lib, maybe not from the repo) as I really don't want people to rely on those components. They are meant to illustrate and test how the entity decorators are created, but I would expect users of the lib to make their own.